config/lava/chromeos/chromeos-flash-template.jinja2

```diff
@@ -50,35 +50,12 @@ actions:
         run:
           steps:
             - ping -c 1 172.17.0.1
-            - lava-test-case wget --shell  "wget http://172.17.0.1/chromeos/{{ chromeos_image }}"
-            - lava-test-case scp --shell "scp -i /keys/id_rsa_chromeos_flash -o ConnectTimeout=3 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {{ chromeos_image }} root@$(lava-target-ip):/root"
-      from: inline
-      name: docker-network
-      path: inline/docker-network.yaml
-
-- test:
-    timeout:
-      minutes: 60
-    definitions:
-    - repository:
-        metadata:
-          format: Lava-Test Test Definition 1.0
-          name: chromeos-flash
-          description: "Flash Chrome OS image"
-          os:
-            - debian
-          scope:
-            - functional
-          environment:
-            - lava-test-shell
-        run:
-          steps:
-            - lsblk
-            - ls -l /root
-            - IMAGE_NAME={{ chromeos_image }}  /bin/bash /opt/chromeos/flash
-      lava-signal: kmsg
+            - lava-test-case wget --shell "wget --no-check-certificate -O chromiumos_test_image.bin.gz {{ chromeos_image }}"
+            - lava-test-case unpack --shell "gzip -d chromiumos_test_image.bin.gz || mv chromiumos_test_image.bin.gz chromiumos_test_image.bin"
+            - lava-test-case scp --shell "scp -C -i /keys/id_rsa_chromeos_flash -o ConnectTimeout=3 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null chromiumos_test_image.bin root@$(lava-target-ip):/root"
+            - ssh -i /keys/id_rsa_chromeos_flash -o ConnectTimeout=3 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$(lava-target-ip) "IMAGE_NAME=chromiumos_test_image.bin FLASH_DEVICE={{ flashing_device }} /bin/bash /opt/chromeos/flash"
       from: inline
       name: chromeos-flash
-      path: inline/chromeos-flash.yaml
```

Comment:
Looks like this was not intended?

Reply:
The "repository" name is also still called `docker-network` instead of `chromeos-flash` (but I can't put a comment as it's outside the diff).

Reply:
This whole section is removed, because i changed flashing from serial to ssh.
As we have hanging console issue, ssh is way more reliable.

Reply:
I will update names in patch to make it less confusing

Reply:
Now i fixed both patches properly. I removed whole section chromeos-flash that works over console, and added "             - ssh -i /keys/id_rsa_chromeos_flash -o ConnectTimeout=3 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$(lava-target-ip) "IMAGE_NAME=chromiumos_test_image.bin FLASH_DEVICE={{ flashing_device }} /bin/bash /opt/chromeos/flash"" to docker-network, so it does flashing too, and to remove confusion renamed it to docker-network-flashing

Reply:
But it should just be `chromeos-flash`, right? The `docker-network` thing is just some leftover from initial debugging things to check whether Docker on the dispatcher could reach the Chromebook.